aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -984,6 +984,34 @@ Tensor hann_window(
       window_length, periodic, /*alpha=*/0.5, /*beta=*/0.5, options);
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ kaiser_window ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor kaiser_window(int64_t window_length, double beta, const TensorOptions& options) {
+  return native::kaiser_window(window_length, beta, /*periodic=*/true, options);
```

Comment:
Why include the periodic arg? Following https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.windows.kaiser.html?highlight=kaiser#scipy.signal.windows.kaiser, wouldn't it be more natural for the arg to be "sym" with a default of True? It's also fine to not expose this argument for now if it's unsupported.